github/MainClass.py

```diff
@@ -769,7 +773,9 @@ def create_from_raw_data(
         if headers is None:
             headers = {}
 
-        return klass(self.__requester, headers, raw_data, completed=True)
+        if issubclass(klass, CompletableGithubObject):
+            return klass(self.__requester, headers, raw_data, completed=True)  # type: ignore
```

Comment:
You've been removing `completed` from many constructors, why is it still needed here?

Reply:
This can probably be removed as well.